src/components/AboutUs/AboutUsModal/index.js

```diff
@@ -20,21 +20,18 @@ class AboutUsModal extends React.Component {
         <Modal.Body>
           <h4>DoGether</h4>
           <p>
-            DoGether is a task management web app developed by four UBC
-            students. Its aim is to add a social aspect to task management in
-            order to help users motivate users.
+            <br />
+            DoGether is a social task management app developed by four UBC
+            students which helps users motivate users.
```

Comment:
How about, "DoGether is a social task management app developed by four UBC students that aims to help users motivate users."

Reply:
sure lets do that